tools/Vis_PV.py

```diff
@@ -0,0 +1,490 @@
+#!/usr/bin/env python
```

Comment:
What do you think of the file name `VisParaView.py`? Then we avoid possible confusion for other people who are not familiar with PV==ParaView, and we follow the pascal case convention for file names :) Feel free to suggest something different!